### PR TITLE
#161404824 Add a readme and feedback bug fixes

### PR DESCRIPTION
**Pull request by kwanj-k**
### What does this PR do?
Add a readme for the repo

#### Description of Task to be Completed?
-- Add a repo read
-- Remove abort and use return 

### How should this be manually tested?
* After cloning the repo, git pull the branch ch-readme-#161404824
* Create a virtual enviroment and activate it : :
    --- virtualenv -p python3 venv
   
    --- source venv/bin/activate

* Run the following:
  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```

Also, you need to create a development and test database and connect to it.[Refer here](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-18-04#creating-a-new-role)

Then export the database URL in the environment:
```
export Dev_URL="dbname=dbname user=username password=password"
export Test_URL="dbname=dbname user=username password=password"
export SECRET="iamasecrettheunsadtheunsulliedtheundead"

```
We need the secret key for generating the tokens

### What are the relevant Pivotal Tracker stories?
  #161523838

### CODE REVIEW

File: app/api/common/validators.py [hunk 1]
```diff
@@ -20,17 +20,18 @@ def common(l, d):
...
-            abort(406, msg)
+            return {"message":msg},400
     for i, v in d.items():
         if not isinstance(v, int):
             gv = "".join(v.split())
             if gv == "":
                 msg = 'The {} can not be empty'.format(i)
-                abort(406, msg)
+                return {"message":msg},400
+                #abort(400,msg)
```
Comment (Collins33) [inline app/api/common/validators.py]: Remove commented out code

File: app/api/common/validators.py [hunk 2]
```diff
@@ -39,51 +40,59 @@ def commonp(d):
         if i == 'name':
```
Comment (Collins33) [inline app/api/common/validators.py]: Use descriptive names for variables

**Review by Collins33 — CHANGES_REQUESTED** [review]

File: app/api/common/validators.py (hunk 1 above)
Comment (kwanj-k) [reply app/api/common/validators.py]: done

File: app/api/common/validators.py (hunk 2 above)
Comment (kwanj-k) [reply app/api/common/validators.py]: done

**Pull request merged**